clang/docs/ReleaseNotes.rst

```diff
@@ -918,6 +918,10 @@ Bug Fixes to C++ Support
   (`#57410 <https://github.com/llvm/llvm-project/issues/57410>`_) and
   (`#76604 <https://github.com/llvm/llvm-project/issues/57410>`_)
 
+- Emit an error when the same identifier appears with both internal and
+  external linkage in a translation unit Fixes:
```

Comment:
```suggestion
  external linkage in a translation unit. Fixes:
```
I think the release note should also be moved to be under `Bug Fixes in This Version` because it's not a C++-specific diagnostic.